DOL.WHD.Section14c.Web/src/modules/components/sectionWageData/wageDataPayTypeFormTemplate.html

```diff
@@ -0,0 +1,291 @@
+                    <label for="wagemethod_survey">Prevailing Wage Survey</label>
+                </li>
+                <li>
+                    <input id="wagemethod_alternate" type="radio" name="wage-method" value="alternate" ng-model="formData.prevailingWageMethod">
+                    <label for="wagemethod_alternate">Alternate Wage Data</label>
+                </li>
+                <li>
+                    <input id="wagemethod_sca" type="radio" name="wage-method" value="sca" ng-model="formData.prevailingWageMethod">
+                    <label for="wagemethod_sca">SCA Wage Determination</label>
+                </li>
+            </ul>
+        </fieldset>
+    </div>
+
+    <div class="form-question-group em-block" ng-show="formData.prevailingWageMethod === 'survey'">
+        <div class="form-question-block">
+            <div class="form-question-text">Provide information for the most recent Prevailing Wage survey conducted for the job or contract indentified above:</div>
+            <div class="form-question-answer">
+                <label for="prevailingWageDetermined">Prevailing Wage Determined based on this survey</label>
+                <input id="prevailingWageDetermined" name="prevailingWageDetermined" type="number" class="wage-rate" min="0" step="0.01">
+                <label>You must provide <strong>3</strong> Source Employers for the survey</label>
+                <div ng-show="vm.addingEmployer">
+                    <hr>
+                    <label for="sourceEmployerName">Source Employer Name</label>
+                    <input id="sourceEmployerName" name="sourceEmployerName" type="text" ng-model="vm.activeSourceEmployer.employerName">
+
+                    <form class="usa-form-large">
+                        <fieldset>
+                          <label for="mailing-address-1">Street address 1</label>
+                          <input id="mailing-address-1" name="mailing-address-1" type="text" ng-model="vm.activeSourceEmployer.address.streetAddress">
+
+                          <label for="mailing-address-2">Street address 2 <span class="usa-additional_text">Optional</span></label>
+                          <input id="mailing-address-2" name="mailing-address-2" type="text">
+
+                          <div>
+                            <div class="usa-input-grid usa-input-grid-medium">
+                              <label for="city">City</label>
+                              <input id="city" name="city" type="text" ng-model="vm.activeSourceEmployer.address.city">
+                            </div>
+
+                            <div class="usa-input-grid usa-input-grid-small">
+                              <label for="state">State</label>
+                              <select id="state" name="state" ng-model="vm.activeSourceEmployer.address.state">
+                                <option value></option>
+                                <option value="AL">Alabama</option>
```

Comment:
I will add those in (I pulled the current values straight from the US WDS website). Is there an "official" list off all the territories to include? Is it just Puerto Rico, American Samoa, Guam, Northern Mariana Islands, and the U.S. Virgin Islands? Any other?